docs/yuidoc-p5-theme/assets/reference.css

```diff
@@ -0,0 +1,23 @@
+/**
+ * Note: most of the styling for the reference documentation is actually
+ * contained in the p5.js-website repository:
+ *
+ * https://github.com/processing/p5.js-website/blob/master/css/main.css
+ *
+ * However, some of it is located here so we can easily iterate on it
+ * in the same repository.
```

Comment:
Um, sorry if this is a terrible idea :sweat:

Would it be useful to actually move all the reference-related CSS from [p5.js-website's `main.css`](https://github.com/processing/p5.js-website/blob/master/css/main.css) over here, though? Or is some of that reference CSS used elsewhere on the p5.js website?